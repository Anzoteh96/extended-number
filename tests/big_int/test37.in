2580165864587222817692347799118547338996062764647792057856669127029785609436245712307671531695126861076656739184
728803349599644902184541381275400252313968285192716624903356794716899395684468029183679042336956745694644217450
