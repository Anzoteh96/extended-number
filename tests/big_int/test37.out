Enter two numbers n and m:
n is 2580165864587222817692347799118547338996062764647792057856669127029785609436245712307671531695126861076656739184
m is 728803349599644902184541381275400252313968285192716624903356794716899395684468029183679042336956745694644217450
n + m is 3308969214186867719876889180393947591310031049840508682760025921746685005120713741491350574032083606771300956634
n - m is 1851362514987577915507806417843147086682094479455075432953312332312886213751777683123992489358170115382012521734
n * m is 1880433524633831799633697397208231734678592234135667881213553642343964146221206324811472433894821882846773419941601483249606042456842972717362738902878207554528660522570616485723664110901550988622702436823681132738431560800
n / m is 3
n % m is 393755815788288111138723655292346582054157909069642183146598742879087422382841624756634404684256623992724086834
